quick-start/examples/n9.min.js

```diff
@@ -0,0 +1,50 @@
+import { Scene, Circle, Rectangle, Color, RadialGradient, Star, Particles }
+    from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: new Color(),
+});
+const { width, height } = scene;
+scene.cursorPosition.set(scene.center).subtract(0, height / 4);
+
+const sun = new Circle(scene.center.add(0, height / 2), 100, {
+    fill: "orange",
+});
+
+const ocean = new Rectangle([0, height / 2], width, height / 2, {
+    fill: new RadialGradient(undefined, sun.radius * 3, {
+        0: sun.options.fill,
+        1: "#28488f",
+    }),
+});
+
+// Base star shape used by all particles
+const base = new Star(undefined, 6, 4, 0.2, {
+    fill: "white",
+});
+const numberOfStars = 150;
+// Function that generate individual particle options
+const generator = () => ({
+    position: scene.getRandomPosition().divide(1, 2),
+    scale: Math.random(),
+});
+// Particles object
+const stars = new Particles([0, 0], base, numberOfStars, generator);
+
+scene
+    .add(stars, sun, ocean)
+    .startLoop()
+    .on("draw", () => {
+        sun.position.lerp(scene.cursorPosition, 0.05);
+
+        const setting = (sun.position.y - (height / 4)) / (height / 2);
+        scene.options.fill
+            .set("#45a8ff")
+            .lerp("#150a1b", setting);
+
+        ocean.options.fill.position
+            .set(sun.position.x, height - sun.position.y - (height / 2));
+
+        // Update stars opacity
+        stars.options.opacity = setting ** 2;
```

Comment:
The ES2016 `**` Exponentiation should be explained. Many beginners and intermediate developers will not know this one. (I had to look it up.)

Reply:
I didn't want people to stop on the details, but if you found this confusing, I'll add a comment.